tests/conftest.py

```
@@ -112,9 +121,32 @@ def getRandomValues(buf):
         return b"\x00" * 12
 
 
+class _WebSocketPair:
+    """Stub for js.WebSocketPair."""
+
+    @staticmethod
+    def new():
```

Ruff (0.15.10)

[warning] Missing return type annotation for staticmethod `new`

(ANN205)